packages/blade/src/components/Input/BaseInput/BaseInput.stories.tsx

```diff
@@ -4,25 +4,155 @@ import { Highlight } from '@storybook/design-system';
 import React from 'react';
 import type { BaseInputProps } from './BaseInput';
 import { BaseInput as BaseInputComponent } from './BaseInput';
+import iconMap from '~components/Icons/iconMap';
+import { CloseIcon } from '~components/Icons';
 
 export default {
   title: 'Components/Input/BaseInput (Internal)',
   component: BaseInputComponent,
   args: {
     id: 'base-input',
     defaultValue: undefined,
-    label: 'Enter Name',
     placeholder: 'Enter your first and last name',
-    labelPosition: 'top',
     name: 'fullName',
     type: 'text',
     isDisabled: false,
+    value: undefined,
+    maxCharacters: 9999,
     onChange: undefined,
+    label: 'Enter Name',
+    labelPosition: 'top',
+    neccessityIndicator: undefined,
+    isRequired: false,
     validationState: 'none',
     helpText: undefined,
     errorText: undefined,
     successText: undefined,
-    isRequired: false,
+    leadingIcon: undefined,
+    prefix: '',
+    interactionElement: <CloseIcon size="small" color="surface.text.subtle.lowContrast" />,
+    suffix: '',
+    trailingIcon: undefined,
+  },
+  argTypes: {
+    id: {
+      table: {
+        category: 'Base Input Props',
+      },
+    },
+    defaultValue: {
+      table: {
+        category: 'Base Input Props',
+      },
+    },
+    placeholder: {
+      table: {
+        category: 'Base Input Props',
```

Comment:
Let's extract the groups to constants and then use it. 

```
const BASE_PROPS = 'Base Input Props';
```

Reply:
fixed in Pr-4